src/mixins/filing-template-mixin.ts

```diff
@@ -290,24 +292,36 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
         identifier: this.getEntitySnapshot.businessInfo.identifier,
         legalName: this.getEntitySnapshot.businessInfo.legalName,
         legalType: this.getEntitySnapshot.businessInfo.legalType,
-        nrNumber: this.getEntitySnapshot.businessInfo.nrNumber,
-        associationType: this.getAssociationType
+        nrNumber: this.getEntitySnapshot.businessInfo.nrNumber
       },
       specialResolution: {
-        ...this.getcreateResolution
+        ...this.getSpecialResolution
+      },
+      alteration: {
```

Comment:
so I think this should be conditional like change of name.  for example, if there was only a change of name, then you would have a SR + CoN filing.  Is this taken care of already and I'm just seeing it?

Reply:
I've made it conditional